common/config/azure-pipelines/jobs/docs-build.yaml

```diff
@@ -76,7 +76,11 @@ jobs:
           useCurrentBisDocsArtifact: ${{ parameters.useCurrentBisDocsArtifact }}
 
       # Currently BeMetalsmith is an internal only tool
+<<<<<<< HEAD
```

Comment:
@ben-polinsky Is the 4.3.x that was already here correct for 3.7?

Reply:
I'm assuming it was, and resolving the merge conflict accordingly.

Reply:
Thanks correct assumption.